ledger/api-server-damlonx/BUILD.bazel

```diff
@@ -41,5 +41,6 @@ da_scala_library(
         "//ledger/ledger-api-scala-logging",
         "//ledger/participant-state",
         "//ledger/participant-state-index",
+        "//ledger/participant-state-index:participant-state-index-v2",
```

Comment:
daml-on-x shouldn't use the v2 index api.

Reply:
Only `v2.Identityservice` is needed. Could we keep it?

Reply:
nothing of v2 participant-state-index should be used in damlonx stuff.

Reply:
I massaged the code so we can get it away without it.